server/server.go

```diff
@@ -405,6 +409,16 @@ func (s *Server) StartExitMonitor() {
 							}
 						}
 					}
+					if c != nil && s.watcherWaitContainer(containerID) {
+						logrus.Debugf("waiting for container or sandbox: %v", containerID)
+						go func() {
+							if err := s.Runtime().PollUpdateStatusStopped(c, 10); err != nil {
+								logrus.Warnf("container state not updated to 'stopped' %v: %v", containerID, err)
+								return
+							}
+							logrus.Debugf("container state updated to 'stopped' %v: %v", containerID, err)
```

Comment:
Do we want an `s.ContainerStateToDisk(c)` here?

Reply:
This will be done from the StopContainer() waiting for the in-memory state to happen. I mean I don't think this would hurt to add an extra call to `s.ContainerStateToDisk(c)`, but I don't think it is really useful either.